packages/camera/camera_avfoundation/ios/Classes/FLTCam.h

```diff
@@ -95,17 +93,26 @@ NS_ASSUME_NONNULL_BEGIN
 /// @param focusMode The focus mode that should be applied to the @captureDevice instance.
 /// @param captureDevice The AVCaptureDevice to which the @focusMode will be applied.
 - (void)applyFocusMode:(FCPPlatformFocusMode)focusMode onDevice:(AVCaptureDevice *)captureDevice;
-- (void)pausePreviewWithResult:(FlutterResult)result;
-- (void)resumePreviewWithResult:(FlutterResult)result;
-- (void)setDescriptionWhileRecording:(NSString *)cameraName result:(FlutterResult)result;
-- (void)setExposurePointWithResult:(FlutterResult)result x:(double)x y:(double)y;
-- (void)setFocusPointWithResult:(FlutterResult)result x:(double)x y:(double)y;
-- (void)setExposureOffsetWithResult:(FlutterResult)result offset:(double)offset;
+- (void)pausePreview;
+- (void)resumePreview;
+- (void)setDescriptionWhileRecording:(NSString *)cameraName
+                      withCompletion:(void (^)(FlutterError *_Nullable))completion;
+
+/// Sets the exposure point, in a (0,1) coordinate system.
+///
+/// If @c point is nil, the exposure point will reset to the center.
```

Comment:
This logic was in CameraPlugin, but I moved it into this class now that it was easy to pass nil through using the new class.